src/main/java/xbot/common/controls/actuators/mock_adapters/MockCANTalon.java

```diff
@@ -123,7 +123,11 @@ public void set(ControlMode Mode, double demand0, double demand1) {
     }
 
     public double getThrottlePercent() {
-        return this.throttlePercent;
+        if(master == null){
```

Comment:
Looks like this was fixing a bug - I see that there's some follower logic in the `set()` method, but I guess it wasn't being called properly?

In any case, this seems sensible.